target/scripts/startup/setup-stack.sh

```diff
@@ -768,22 +786,10 @@ function _setup_docker_permit
       echo 172.16.0.0/12 >> /etc/opendkim/TrustedHosts
       ;;
 
-    "connected-networks" )
-      for NETWORK in "${CONTAINER_NETWORKS[@]}"
-      do
-        NETWORK=$(_sanitize_ipv4_to_subnet_cidr "${NETWORK}")
-        _notify 'inf' "Adding docker network ${NETWORK} in my networks"
-        postconf -e "$(postconf | grep '^mynetworks =') ${NETWORK}"
-        echo "${NETWORK}" >> /etc/opendmarc/ignore.hosts
-        echo "${NETWORK}" >> /etc/opendkim/TrustedHosts
-      done
-      ;;
-
     * )
-      _notify 'inf' 'Adding container ip in my networks'
-      postconf -e "$(postconf | grep '^mynetworks =') ${CONTAINER_IP}/32"
-      echo "${CONTAINER_IP}/32" >> /etc/opendmarc/ignore.hosts
-      echo "${CONTAINER_IP}/32" >> /etc/opendkim/TrustedHosts
+      _notify 'warn' "Invalid value for PERMIT_DOCKER: ${PERMIT_DOCKER}"
+      _notify 'inf' "Clearing Postfix's 'mynetworks'"
+      postconf -e "mynetworks ="
```

Comment:
This happens with the default `none` I take it? So anyone who was relying on previous behaviour will see the warning at least in their logs?

Reply:
No. `PERMIT_DOCKER=` now defaults to `PERMIT_DOCKER=none`.

This warning will only be shown, if something unexpected is set, e.g. a typo like `PERMIT_DOCKER=hosts`

Reply:
Yup, edited my comment while reviewing and realized I misunderstood :grimacing: 

Users will have to rely on checking release notes before upgrading or raising issues to find out if the minor update impacts them. Shouldn't be a problem, this is a project you shouldn't blindly update AFAIK :+1: